ui/packages/consul-ui/translations/en-us.yaml

```diff
@@ -6,11 +6,14 @@ common:
     vault: Vault
     aws: AWS
     kubernetes: Kubernetes
+    jwt: JWT
+    oidc: OIDC
   ui:
     remove: Remove {item}
     filtered-by: Filtered by {item}
     name: Name
     creation: Creation
+    maxttl: Max TTL
```

Comment:
I think this is fine here for now, but just so you know what I'm thinking - I think this is the kind of thing we could potentially put on a top level var:

```yaml
models:
  authmethod:
    maxttl: MaxTLL
    tokenlocality: Token Locality
etc etc...
```

You might want to do this for view pages PR.

Reply:
ah, yes. I'll keep a notes of this. Thanks!